GCC6 -marm build error

Running into a compilation error on Lakka....

```
  UNPACK   beetle-pcfx
Cloning into '/root/build.Lakka-Generic.x86_64-2.1-devel/beetle-pcfx-111625e'...
remote: Counting objects: 6610, done.
remote: Compressing objects: 100% (133/133), done.
remote: Total 6610 (delta 85), reused 132 (delta 61), pack-reused 6406
Receiving objects: 100% (6610/6610), 24.38 MiB | 375.00 KiB/s, done.
Resolving deltas: 100% (3492/3492), done.
Checking connectivity... done.
Note: checking out '111625e'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 111625e... Add missing file
  BUILD    beetle-pcfx (target)
make[1]: Entering directory '/root/build.Lakka-Generic.x86_64-2.1-devel/beetle-pcfx-111625e'
/root/build.Lakka-Generic.x86_64-2.1-devel/toolchain/bin/x86_64-libreelec-linux-gnu-g++ -c -o mednafen/pcfx/king.o mednafen/pcfx/king.cpp -march=x86-64 -m64 -fomit-frame-pointer -Wall -pipe -O2 -flto -ffat-lto-objects -mmmx -msse -msse2 -mfpmath=sse -DGIT_VERSION=\"" 111625e"\" -pthread -marm -DARM -DWANT_THREADING -DWANT_32BPP -DWANT_NEW_API -DWANT_STEREO_SOUND -DFRONTEND_SUPPORTS_RGB565 -DNEED_CD -DHAVE_CHD -D_7ZIP_ST -DPACKAGE_VERSION=\"1.3.2\" -DFLAC_API_EXPORTS -DFLAC__HAS_OGG=0 -DHAVE_LROUND -DHAVE_STDINT_H -DHAVE_STDLIB_H -DHAVE_SYS_PARAM_H -DNEED_TREMOR -O2  -fPIC  -I. -I./mednafen -I./mednafen/include -I./mednafen/intl -I./mednafen/hw_sound -I./mednafen/hw_cpu -I./mednafen/hw_misc -I./libretro-common/include -I./deps/zlib -I./libretro-common/include -I./deps/crypto -I./deps/flac-1.3.2/include -I./deps/flac-1.3.2/src/libFLAC/include -I./deps/lzma-16.04/C -I./deps/libchdr  -DSIZEOF_DOUBLE=8 -Wall -Wno-sign-compare -Wno-unused-variable -Wno-unused-function -Wno-uninitialized  -Wno-strict-aliasing -DMEDNAFEN_VERSION=\"0.9.36.5\" -DPACKAGE=\"mednafen\" -DMEDNAFEN_VERSION_NUMERIC=9365 -DPSS_STYLE=1 -DMPC_FIXED_POINT -DWANT_PCFX_EMU -DSTDC_HEADERS -D__STDC_LIMIT_MACROS -D__LIBRETRO__ -D_LOW_ACCURACY_  
/root/build.Lakka-Generic.x86_64-2.1-devel/toolchain/bin/x86_64-libreelec-linux-gnu-g++ -c -o mednafen/pcfx/soundbox.o mednafen/pcfx/soundbox.cpp -march=x86-64 -m64 -fomit-frame-pointer -Wall -pipe -O2 -flto -ffat-lto-objects -mmmx -msse -msse2 -mfpmath=sse -DGIT_VERSION=\"" 111625e"\" -pthread -marm -DARM -DWANT_THREADING -DWANT_32BPP -DWANT_NEW_API -DWANT_STEREO_SOUND -DFRONTEND_SUPPORTS_RGB565 -DNEED_CD -DHAVE_CHD -D_7ZIP_ST -DPACKAGE_VERSION=\"1.3.2\" -DFLAC_API_EXPORTS -DFLAC__HAS_OGG=0 -DHAVE_LROUND -DHAVE_STDINT_H -DHAVE_STDLIB_H -DHAVE_SYS_PARAM_H -DNEED_TREMOR -O2  -fPIC  -I. -I./mednafen -I./mednafen/include -I./mednafen/intl -I./mednafen/hw_sound -I./mednafen/hw_cpu -I./mednafen/hw_misc -I./libretro-common/include -I./deps/zlib -I./libretro-common/include -I./deps/crypto -I./deps/flac-1.3.2/include -I./deps/flac-1.3.2/src/libFLAC/include -I./deps/lzma-16.04/C -I./deps/libchdr  -DSIZEOF_DOUBLE=8 -Wall -Wno-sign-compare -Wno-unused-variable -Wno-unused-function -Wno-uninitialized  -Wno-strict-aliasing -DMEDNAFEN_VERSION=\"0.9.36.5\" -DPACKAGE=\"mednafen\" -DMEDNAFEN_VERSION_NUMERIC=9365 -DPSS_STYLE=1 -DMPC_FIXED_POINT -DWANT_PCFX_EMU -DSTDC_HEADERS -D__STDC_LIMIT_MACROS -D__LIBRETRO__ -D_LOW_ACCURACY_  
/root/build.Lakka-Generic.x86_64-2.1-devel/toolchain/bin/x86_64-libreelec-linux-gnu-g++ -c -o mednafen/pcfx/interrupt.o mednafen/pcfx/interrupt.cpp -march=x86-64 -m64 -fomit-frame-pointer -Wall -pipe -O2 -flto -ffat-lto-objects -mmmx -msse -msse2 -mfpmath=sse -DGIT_VERSION=\"" 111625e"\" -pthread -marm -DARM -DWANT_THREADING -DWANT_32BPP -DWANT_NEW_API -DWANT_STEREO_SOUND -DFRONTEND_SUPPORTS_RGB565 -DNEED_CD -DHAVE_CHD -D_7ZIP_ST -DPACKAGE_VERSION=\"1.3.2\" -DFLAC_API_EXPORTS -DFLAC__HAS_OGG=0 -DHAVE_LROUND -DHAVE_STDINT_H -DHAVE_STDLIB_H -DHAVE_SYS_PARAM_H -DNEED_TREMOR -O2  -fPIC  -I. -I./mednafen -I./mednafen/include -I./mednafen/intl -I./mednafen/hw_sound -I./mednafen/hw_cpu -I./mednafen/hw_misc -I./libretro-common/include -I./deps/zlib -I./libretro-common/include -I./deps/crypto -I./deps/flac-1.3.2/include -I./deps/flac-1.3.2/src/libFLAC/include -I./deps/lzma-16.04/C -I./deps/libchdr  -DSIZEOF_DOUBLE=8 -Wall -Wno-sign-compare -Wno-unused-variable -Wno-unused-function -Wno-uninitialized  -Wno-strict-aliasing -DMEDNAFEN_VERSION=\"0.9.36.5\" -DPACKAGE=\"mednafen\" -DMEDNAFEN_VERSION_NUMERIC=9365 -DPSS_STYLE=1 -DMPC_FIXED_POINT -DWANT_PCFX_EMU -DSTDC_HEADERS -D__STDC_LIMIT_MACROS -D__LIBRETRO__ -D_LOW_ACCURACY_  
/root/build.Lakka-Generic.x86_64-2.1-devel/toolchain/bin/x86_64-libreelec-linux-gnu-g++ -c -o mednafen/pcfx/huc6273.o mednafen/pcfx/huc6273.cpp -march=x86-64 -m64 -fomit-frame-pointer -Wall -pipe -O2 -flto -ffat-lto-objects -mmmx -msse -msse2 -mfpmath=sse -DGIT_VERSION=\"" 111625e"\" -pthread -marm -DARM -DWANT_THREADING -DWANT_32BPP -DWANT_NEW_API -DWANT_STEREO_SOUND -DFRONTEND_SUPPORTS_RGB565 -DNEED_CD -DHAVE_CHD -D_7ZIP_ST -DPACKAGE_VERSION=\"1.3.2\" -DFLAC_API_EXPORTS -DFLAC__HAS_OGG=0 -DHAVE_LROUND -DHAVE_STDINT_H -DHAVE_STDLIB_H -DHAVE_SYS_PARAM_H -DNEED_TREMOR -O2  -fPIC  -I. -I./mednafen -I./mednafen/include -I./mednafen/intl -I./mednafen/hw_sound -I./mednafen/hw_cpu -I./mednafen/hw_misc -I./libretro-common/include -I./deps/zlib -I./libretro-common/include -I./deps/crypto -I./deps/flac-1.3.2/include -I./deps/flac-1.3.2/src/libFLAC/include -I./deps/lzma-16.04/C -I./deps/libchdr  -DSIZEOF_DOUBLE=8 -Wall -Wno-sign-compare -Wno-unused-variable -Wno-unused-function -Wno-uninitialized  -Wno-strict-aliasing -DMEDNAFEN_VERSION=\"0.9.36.5\" -DPACKAGE=\"mednafen\" -DMEDNAFEN_VERSION_NUMERIC=9365 -DPSS_STYLE=1 -DMPC_FIXED_POINT -DWANT_PCFX_EMU -DSTDC_HEADERS -D__STDC_LIMIT_MACROS -D__LIBRETRO__ -D_LOW_ACCURACY_  
x86_64-libreelec-linux-gnu-g++-6.2.0: error: unrecognized command line option '-marm'; did you mean '-mabm'?
x86_64-libreelec-linux-gnu-g++-6.2.0: error: unrecognized command line option '-marm'; did you mean '-mabm'?
make[1]: *** [Makefile:426: mednafen/pcfx/soundbox.o] Error 1
make[1]: *** Waiting for unfinished jobs....
x86_64-libreelec-linux-gnu-g++-6.2.0: error: unrecognized command line option '-marm'; did you mean '-mabm'?
make[1]: *** [Makefile:426: mednafen/pcfx/interrupt.o] Error 1
x86_64-libreelec-linux-gnu-g++-6.2.0: error: unrecognized command line option '-marm'; did you mean '-mabm'?
make[1]: *** [Makefile:426: mednafen/pcfx/king.o] Error 1
make[1]: *** [Makefile:426: mednafen/pcfx/huc6273.o] Error 1
make[1]: Leaving directory '/root/build.Lakka-Generic.x86_64-2.1-devel/beetle-pcfx-111625e'
Makefile:12: recipe for target 'image' failed
make: *** [image] Error 2
```

--- a/Makefile
+++ b/Makefile
@@ -214,7 +214,7 @@ else ifneq (,$(findstring cortexa9,$(platform)))
    FLAGS += -marm -mcpu=cortex-a9
    ASFLAGS += -mcpu=cortex-a9
 endif
-   FLAGS += -marm
+
 ifneq (,$(findstring neon,$(platform)))
    FLAGS += -mfpu=neon
    ASFLAGS += -mfpu=neon

